src/components/Map/index.js

```diff
@@ -23,13 +31,7 @@ const Map = ({ className, mapOptions, hasZoomControls, canBeDragged, children, e
   );
 
   return (
-    <MapComponent
-      className={className}
-      data-testid="map-base"
-      options={options}
-      events={events}
-      setInstance={setInstance}
-    >
+    <StyledMap className={className} data-testid="map-base" options={options} events={events} setInstance={setInstance}>
```

Comment:
This line is longer than the linter should allow

Reply:
No problem here. printWidth is 120 in .prettierrc